things/api/http/requests.go

```diff
@@ -121,14 +138,16 @@ type listResourcesReq struct {
 	token  string
 	offset uint64
 	limit  uint64
+	name   string
 }
 
 func (req *listResourcesReq) validate() error {
 	if req.token == "" {
 		return things.ErrUnauthorizedAccess
 	}
 
-	if req.limit == 0 || req.limit > maxLimitSize {
+	if req.limit == 0 || req.limit > maxLimitSize ||
+		len(req.name) > maxNameSize {
```

Comment:
Please move the name part to a separate `if` statement in order to keep it simple.

Reply:
Yes pls, we are avoiding chaining as much as we can.